flaskbb/user/plugins.py

```diff
@@ -0,0 +1,22 @@
+from itertools import chain
+from pluggy import HookimplMarker
+
+impl = HookimplMarker('flaskbb')
+
+
+@impl(hookwrapper=True, tryfirst=True)
+def flaskbb_tpl_profile_settings_menu():
```

Comment:
I didn't want to fuss with attempting to do both the implementation and the wrapper separately and figuring out how to best load them and instead opted for just combining them.